bin/add-doc-footer.sh

```diff
@@ -19,7 +19,7 @@ function update_footer {
 	printf '%s\n' "<!-- FEEDBACK -->$REPLACEWITH<!-- /FEEDBACK -->" '' >> $1
 }
 
-find ./docs -name "*.md" -or -path "./docs/examples" -prune|while read filename; do
+find ./docs -name "*.md" ! -path "./docs/examples*"  ! -path "./docs/internal-developers*"|while read filename; do
```

Comment:
This now correctly excludes examples and internal developer docs from including the feedback text.